### PR TITLE
add templates for bugs and feature requests

### LINKED ISSUE #1457: configure issue template
I recommend to setup an issue template which contains some important information to the issue reporters like "write in english", "give a descriptive title", "provide version of xDrip", etc.

Good templates can be found at [Awesome GitHub Issues & PRs Templates](https://github.com/devspace/awesome-github-templates).

### PR DESCRIPTION
**Pull request by tolot27**
fixes #1457

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+### If you need support for xDrip, PLEASE DO NOT FILE A TICKET HERE
+For support, please post a question to the xDripG5 group at Facebook (https://www.facebook.com/groups/xDripG5) or visit the Gitter room at https://gitter.im/jamorham/xDrip-plus.
+Please also search the existing issues for similar problems before opening a new one.
+
+### Subject of the issue
+A clear and concise description of what the bug is.
+
+### Your environment
+- What version of xDrip (e.g. 0.10.3)
```
Comment (jamorham) [inline .github/ISSUE_TEMPLATE/bug-report.md:15]: I think it would help if the example was similar to what it would actually be

**Comment by jamorham** [comment]
Other than the comment I made this looks fine. I assume the files are in the correct place to activate on github.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (tolot27) [reply .github/ISSUE_TEMPLATE/bug-report.md]: I've added some xDrip specific example text.

**Comment by tolot27** [comment]
>  I assume the files are in the correct place to activate on github.

Yes, they are in the correct place. I've tested it in my fork: https://github.com/tolot27/xDrip/issues/new/choose

**Review by jamorham — APPROVED** [review]

**Pull request merged**